documentation/modules/configuring/ref-storage-persistent.adoc

```diff
@@ -5,36 +5,48 @@
 [id='ref-persistent-storage-{context}']
 = Persistent storage
 
-Persistent storage uses {K8sPersistentVolumeClaims} to provision persistent volumes for storing data.
-Persistent Volume Claims can be used to provision volumes of many different types, depending on the {K8SStorageClass} which will provision the volume.
-The data types which can be used with persistent volume claims include many types of SAN storage as well as {K8sLocalPersistentVolumes}.
+[role="_abstract"]
+Persistent data storage retains data even when a pod that uses the storage restarts.
+Persistent data storage uses a storage framework called persistent volumes (PV). 
+PVs are storage resources, independent of the pods that use them.
+Pod configuration uses {K8sPersistentVolumeClaims} (PVCs) to make storage requests on the PVs.
+The PVC requests the amount of storage required when a pod is being created.
+The underlying storage infrastructure of the PV does not need to be understood. 
+If a PV matches the storage criteria, the PVC is bound to the PV.
 
-To use persistent storage, the `type` has to be set to `persistent-claim`.
-Persistent storage supports additional configuration options:
+Because of its permanent nature, persistent storage is recommended for production. 
+
+PVCs can request different types of persistent storage by specifying a {K8SStorageClass}.
+Storage classes define storage profiles and dynamically provision PVs.  
+Persistent storage options might include SAN storage types or a {K8sLocalPersistentVolumes}.
+
+To use persistent storage, you set the storage type configuration in the `Kafka` or `ZooKeeper` resource to `persistent-claim`.
+
+Persistent storage also has the following configuration options:
 
 `id` (optional)::
-Storage identification number. This option is mandatory for storage volumes defined in a JBOD storage declaration.
+A storage identification number. This option is mandatory for storage volumes defined in a JBOD storage declaration.
 Default is `0`.
 
 `size` (required)::
-Defines the size of the persistent volume claim, for example, "1000Gi".
+The size of the persistent volume claim, for example, "1000Gi".
 
 `class` (optional)::
 The Kubernetes {K8SStorageClass} to use for dynamic volume provisioning.
```

Comment:
Maybe we could add that additional storage _configuration_ can be usually done as part of the Storage class configuration. For example things such as encryption, iops, file system type (etx4 versus xfs) etc.